### DISCUSSION TITLE
Welcome to ggershaw.github.io Discussions!

### DISCUSSION THREAD

**Discussion by ggershaw** [ggershaw/ggershaw.github.io#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Be nice and don't act like a know it all

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by chrismmilburn** [comment]
Hope your still there Geoff ! 
What is the purpose of having both the FIX50SP2.modified file and the FIX50SP2 file?
